### PR TITLE
Fix Batch Normalization

### PR DESCRIPTION
**Pull request by jvanvugt**
Keras uses `epsilon=1-e3` to avoid division by 0 in batch normalization. Pytorch uses `epsilon=1e-5`. This can cause a major difference in output for large networks. Fortunately, this can easily be fixed when transferring the parameters.

### CODE REVIEW

File: nn_transfer/test/test_architectures.py [hunk 1]
```diff
@@ -17,16 +17,19 @@ def setUp(self):
         self.test_data_unet = np.random.rand(1, 1, 224, 224)
 
     def test_simplenet(self):
+        set_seeds()
```
Comment (gzuidhof) [inline nn_transfer/test/test_architectures.py]: You can use the [`setUp` callback](https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp) so you don't have to repeat it in every test case.

File: nn_transfer/transfer.py [hunk 2]
```diff
@@ -17,6 +17,8 @@
 KERAS_BETA_KEY = 'beta' + VAR_AFFIX
 KERAS_MOVING_MEAN_KEY = 'moving_mean' + VAR_AFFIX
 KERAS_MOVING_VARIANCE_KEY = 'moving_variance' + VAR_AFFIX
+KERAS_EPSILON = 1e-3
```
Comment (gzuidhof) [inline nn_transfer/transfer.py:20]: Do you think we can get these values from Keras/PyTorch itself? That way if they decide to change it, this lib won't break

**Review by gzuidhof — COMMENTED** [review]
Thanks a lot Joris :), left some comments for small improvements

File: nn_transfer/transfer.py (hunk 2 above)
Comment (jvanvugt) [reply nn_transfer/transfer.py:20]: Both are hardcoded default parameters. If you really want  to you can do
```python
import torch
import inspect
sig = inspect.signature(torch.nn.functional.batch_norm)
PYTORCH_EPSILON = sig.parameters['eps'].default
```
But then that breaks when they change the function/parameter name

**Comment by gzuidhof** [comment]
Thank you so much again :). Is this ready to merge?

**Comment by jvanvugt** [comment]
He is ready

**Comment by gzuidhof** [comment]
Grazie
